### PR TITLE
[zh] Fix the etcd client certs

### PR DESCRIPTION
**Pull request by h4ghhh**
Same as pr https://github.com/kubernetes/website/pull/30562

### CODE REVIEW

**Review by Arhell — COMMENTED** [review]
/assign @tengqm

File: content/zh/docs/tasks/administer-cluster/configure-upgrade-etcd.md [hunk 1]
```diff
@@ -665,4 +665,4 @@ ETCDCTL_API=3 etcdctl --endpoints 10.2.0.9:2379 snapshot restore snapshotdb
 我们还建议重启所有组件（例如 `kube-scheduler`、`kube-controller-manager`、`kubelet`），以确保它们不会
 依赖一些过时的数据。请注意，实际中还原会花费一些时间。
 在还原过程中，关键组件将丢失领导锁并自行重启。
-{{< note >}}
\ No newline at end of file
+{{< note >}}
```
Comment (tengqm) [inline content/zh/docs/tasks/administer-cluster/configure-upgrade-etcd.md:668]:
```suggestion
{{< /note >}}
```

**Comment by tengqm** [comment]
There is another change which needs to be resync'ed.

````
$ ./scripts/lsync.sh content/zh/docs/tasks/administer-cluster/configure-upgrade-etcd.md
diff --git a/content/en/docs/tasks/administer-cluster/configure-upgrade-etcd.md b/content/en/docs/tasks/administer-cluster/configure-upgrade-etcd.md
index ffa688e1b..e1693a9ba 100644
--- a/content/en/docs/tasks/administer-cluster/configure-upgrade-etcd.md
+++ b/content/en/docs/tasks/administer-cluster/configure-upgrade-etcd.md
@@ -325,6 +325,10 @@ Here is an example:
 ```shell
 ETCDCTL_API=3 etcdctl --endpoints 10.2.0.9:2379 snapshot restore snapshotdb
 ```
+Another example for restoring using etcdctl options:
+```shell
+ETCDCTL_API=3 etcdctl --data-dir <data-dir-location> snapshot restore snapshotdb
+```

 For more information and examples on restoring a cluster from a snapshot file, see
 [etcd disaster recovery documentation](https://etcd.io/docs/current/op-guide/recovery/#restoring-a-cluster).
````

**Comment by h4ghhh** [comment]
> ./scripts/lsync.sh content/zh/docs/tasks/administer-cluster/configure-upgrade-etcd.md

Ok, I will fix this in another PR.

**Pull request merged**